src/features/lessons/list-of-lessons/list-of-lessons.js

```diff
@@ -65,18 +63,58 @@ export const ListOfLessons = () => {
     setVisibleLessonsList(filteredLessonsList.slice(indexOfFirst, indexOfLast));
   }, [currentPage, filteredLessonsList]);
 
-  const handleSearchTheme = (inputValue) => setSearchLessonsThemeValue(inputValue);
+  function getYearMonthDayFormat(inputFormat) {
+    function pad(s) { return (s < 10) ? '0' + s : s; }
+    const d = new Date(inputFormat)
+    return [pad(d.getFullYear()), pad(d.getMonth()+1), d.getDate(), ].join('-')
+  }
```

Comment:
Why wouldn't you use commonHelpers.transformDateTime({}).reverseDate to get the same result?

Reply:
fixed with new commit